.eslintrc.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    extends: ['./node_modules/@dhis2/cli-style/config/js/eslint.config.js'],
```

Comment:
I was thinking, to prevent having to deep import configs, we could publish our configs separately on npm, either per tool or all together. Or if we don't want to do that, export them from cli-style as `configs`. So that you could do:

```
module.exports = {
  extends: ["our-eslint-config"]
}
```

Or:

```
const { configs } = require("@dhis2/cli-style");

module.exports = {
  extends: [configs.eslint]
}
```

Publishing the configs separately per tool seems like the most straightforward to me. What do you think?

Reply:
@ismay Good idea. Both of them. 👍 

I had two attempts at turning cli-style into a monorepo that had the tool in one package and one package per configuration. I'm not super-excited about that since we already have 12 configuration files which in theory then would be pulled out into a separate package.

Maybe we do want this eventually, but it is harder stepping back from that approach than stepping into it. If we publish the configs as a completely separate package, it would still need deep imports, or the second approach (`const { configs } = require('@dhis2/cli-style')`) so let's try that first while keeping it all in a single package.

Reply:
Yeah second approach sounds great

Reply:
> Maybe we do want this eventually, but it is harder stepping back from that approach than stepping into it.

Yeah makes sense. Bundling all the configs together seems fine to me 👍

Reply:
https://github.com/dhis2/cli-style/pull/93/files